resources/views/layouts/_twitter_embeds.blade.php

```diff
@@ -0,0 +1,30 @@
+@if ($appId = config('services.twitter.embeds.enabled'))
+<script>
+    (function ($, block) {
+        function embedTwitterWidgetScript() {
+            var notEmbedded = $('script[src="//platform.twitter.com/widgets.js"]').length === 0;
+            if (notEmbedded) {
+                $('body').append('<script async src="//platform.twitter.com/widgets.js" charset="utf-8"><\/script>');
```

Comment:
We can use `@stack` for this instead.

Add the following just before the closing head tag in the `base.blade.php` file with a white line above it:

```php
@stack('scripts-head')
```

Add the following above the content snippet in the `forum/threads/show.blade.php` file: 

```php
@push('scripts')
    <script async src="//platform.twitter.com/widgets.js" charset="utf-8"><\/script>
@endpush
```
This way the script only gets rendered on the thread page.